src/mlx/jira_juggler/jira_juggler.py

```diff
@@ -594,8 +594,36 @@ def load_issues_from_jira(self, depend_on_preceding=False, sprint_field_name='',
             try:
                 issues = jirahandle.search_issues(self.query, maxResults=JIRA_PAGE_SIZE, startAt=self.issue_count,
                                                   expand='changelog')
-            except JIRAError:
-                logging.error('Invalid Jira query "%s"', self.query)
+            except JIRAError as err:
+                logging.error('Failed to query JIRA: %s', err)
+                if err.status_code == 401:
+                    logging.error('Please check your JIRA credentials in the .env file or environment variables.')
+                elif err.status_code == 403:
+                    logging.error('You do not have permission to access this JIRA project or query.')
+                elif err.status_code == 404:
+                    logging.error('The JIRA endpoint is not found. Please check the endpoint URL.')
+                elif err.status_code == 400:
+                    # Parse and display the specific JQL errors more clearly
+                    try:
+                        error_data = err.response.json()
+                        if 'errorMessages' in error_data:
+                            for error_msg in error_data['errorMessages']:
+                                logging.error('JIRA query error: %s', error_msg)
+
+                            # Provide specific guidance based on common errors
+                            for error_msg in error_data['errorMessages']:
```

Comment:
```suggestion
                            # Provide specific guidance based on common errors
                            for error_msg in error_data['errorMessages']:
                                logging.error('JIRA query error: %s', error_msg)

```
These loops are the same, so they could be combined?

Reply:
```suggestion
                            # Provide specific guidance based on common errors
                            for error_msg in error_data['errorMessages']:
                                logging.error(f'JIRA query error: {error_msg}')
```
Or using the f-string

Reply:
The order of the logs was important. I found the extra logs unhelpful and removed them in f0b8d4ae03d48ed15f5e5625777e75123a5fcd9b.